azure/terraform/databricks.rego

```diff
@@ -0,0 +1,119 @@
+package rule
+
+# https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace
+
+#
+# PR-AZR-0070-TRF
```

Comment:
@rezoan  please use the same number for the Azure Databricks in ARM template and in Terraform

Reply:
done.